### ISSUE TITLE
Keep node highlighted by clicking, or alt-click

### ISSUE THREAD

**Issue by rmflight** [timelyportfolio/RBioFabric#2]
Is it possible to keep a node highlighted by selecting it, or using alt-select, so that can easily unzoom and see all the other things it intersects with?

**Comment by timelyportfolio** [comment]
Good idea.  Shouldn't be too hard to implement once I find a sliver of time.  Thanks!
